gam/kendall_tau_distance.py

```diff
@@ -147,10 +147,15 @@ def mergeSortDistance(r1, r2):
     return dist
 
 
-def pairwise_distance_matrix(rankings):
+def pairwise_distance_matrix(rankings, dask=False):
     from sklearn.metrics import pairwise_distances
-
-    D = pairwise_distances(rankings, metric=mergeSortDistance)
+    import numpy as np
+    from dask_ml.metrics.pairwise import pairwise_distances as dask_pairwise_distances
+    import dask.array as da
```

Comment:
This is technically an anti pattern, having imports nested in functions. Usually you would want them at the top of the file.